course/style.css

```diff
@@ -127,6 +127,61 @@ h1, h2, h3, h4, h5, h6 {
 /*
  * Comments
  */
+
+.wp-block-comments-query-loop .wp-block-comments-pagination {
+	margin-top: calc( var(--wp--custom--comment-gap) / 2 * -1 );
+	padding-bottom: var(--wp--custom--comment-gap);
+}
+
+.wp-block-comments-query-loop hr,
+.wp-block-comments-query-loop .wp-block-comments-title {
+	margin-bottom: clamp(2.5rem, 2.167rem + 1.111vw, 3rem);
+}
+
+.wp-block-post-comments-form .comment-form-cookies-consent input[type="checkbox"] {
```

Comment:
How can I test the cookies consent form?

Reply:
It is this checkbox on the bottom that you see when logged out 
<img width="854" alt="image" src="https://github.com/Automattic/themes/assets/6820724/a940a3f6-e3f9-4dfa-90db-f3832dfa87e5">